Added Edge Workspaces to Edge Privacy Whitepaper (#2649)
MicrosoftDocs/edge-developer: microsoft-edge/privacy-whitepaper/index.md

To enable diagnostic data collection, go to `edge://settings/privacy`. Turn on the **Optional diagnostic data** setting. To turn off diagnostic data collection for Microsoft Edge on Windows 10 and later, open **Start** > **Settings** > **Privacy** and select **Diagnostics & feedback**. To turn off diagnostic data collection for all other platforms, go to `edge://settings/privacy`. Turn off the **Help improve Microsoft products by sending optional diagnostic data about how you use the browser, websites you visit, and crash reports** setting. The applications hosting Microsoft Edge WebView may collect other data that is governed by the data collection management of the developer and relevant privacy policies.

<!-- ====================================================================== -->
## Workspaces

> Microsoft Edge Workspaces allows you to easily organize and share your browsing tasks with collaborators via customizable, shareable browsing windows. Each workspace shares a set of tabs, favorites, and history, all created and curated by you and your collaborators. Workspace data is automatically saved, kept up-to-date, and stored in OneDrive/SharePoint.

Reviewer: ```suggestion
The Microsoft Edge Workspaces feature allows you to easily organize and share your browsing tasks with collaborators via customizable, shareable browsing windows. Each workspace shares a set of tabs, favorites, and history, all created and curated by you and your collaborators. Workspace data is automatically saved, kept up-to-date, and stored in OneDrive/SharePoint.
```
alts:
The Microsoft Edge Workspaces feature allows you to
Microsoft Edge Workspaces allows you to 
Microsoft Edge Workspaces allow you to

> The Microsoft Edge Workspaces feature allows you to easily organize and share your browsing tasks with collaborators via customizable, shareable browsing windows. Each workspace shares a set of tabs, favorites, and history, all created and curated by you and your collaborators. Workspace data is automatically saved, kept up-to-date, and stored in OneDrive/SharePoint.

Reviewer: ```suggestion
The Microsoft Edge Workspaces feature allows you to easily organize and share your browsing tasks with collaborators via customizable, shareable browsing windows. Each workspace shares favorites, a set of workspace tabs, and history, all created and curated by you and your collaborators. Workspace data is automatically saved, kept up-to-date, and stored in OneDrive/SharePoint.
```
i found phrasing confusing since i don't know this feature:
a set of a, b, and c -- is the set [a, b, and c]?  or, [a set of a] and also b and c?
Instead, trying phrase from below, "workspace tab".  And different order so it's clear that "set of" only applies to the tabs - if that's the intention.

Or maybe the intention is:
Each workspace shares a set of [tabs, favorites, and history].

Author: The intention is "Each workspace shares a set of [tabs, favorites, and history]." I do see where that phrasing could be confusing, though. How does the following sound, with the removal of "set of"? 

Each workspace shares tabs, favorites, and history.

> Microsoft Edge Workspaces shares tabs, favorites, and history with collaborators in real-time. Collaborators’ profile pictures are used to indicate which workspace tab they are using and which workspace tabs they have opened, changed, and closed. Collaborators cannot see how you interact with a webpage (workspaces do not screen share) and they cannot see password protected content unless they sign in and they have access via their own credentials. Microsoft Edge Workspaces does not store or share browsing data from non-workspace browsing sessions. If you leave a workspace, the stored contents will remain available for other collaborators. If you delete a workspace, the stored contents will be deleted for all collaborators.

Reviewer: ```suggestion
Microsoft Edge Workspaces shares tabs, favorites, and history with collaborators in real-time. Collaborators' profile pictures are used to indicate which workspace tab they are using and which workspace tabs they have opened, changed, and closed. Collaborators cannot see how you interact with a webpage (workspaces do not screen share) and they cannot see password-protected content unless they sign in and they have access via their own credentials. Microsoft Edge Workspaces doesn't store or share browsing data from non-workspace browsing sessions. If you leave a workspace, the stored contents will remain available for other collaborators. If you delete a workspace, the stored contents will be deleted for all collaborators.
```
* straight quote per corp style guide or Docs Contributor Guide
* prefer contractions per corp style guide
* hyphen on adjective pair

> Microsoft Edge Workspaces shares tabs, favorites, and history with collaborators in real-time. Collaborators' profile pictures are used to indicate which workspace tab they are using and which workspace tabs they have opened, changed, or closed. Collaborators cannot see how you interact with a webpage (workspaces do not screen share) and they cannot see password-protected content unless they sign in and they have access via their own credentials. Microsoft Edge Workspaces doesn't store or share browsing data from non-workspace browsing sessions. If you leave a workspace, the stored contents remain available for other collaborators. If you delete a workspace, the stored contents are deleted for all collaborators.

Reviewer: ```suggestion
Microsoft Edge Workspaces shares tabs, favorites, and history with collaborators in real-time. Collaborators' profile pictures are used to indicate which workspace tab they are using and which workspace tabs they have opened, changed, or closed. Collaborators cannot see how you interact with a webpage; workspaces do not screen share. 

Collaborators cannot see password-protected content unless they sign in and they have access via their own credentials. Microsoft Edge Workspaces doesn't store or share browsing data from non-workspace browsing sessions. If you leave a workspace, the stored contents remain available for other collaborators. If you delete a workspace, the stored contents are deleted for all collaborators.
```
* Broke up two separate ideas into two sentences.  Acrolinx would probably suggest breaking up that long sentence.
* Broke up long paragraph.  Readability = short words/phrases/sentences/paragraphs/sections.

> Microsoft Edge Workspaces is only available for enterprise customers who are signed into Microsoft Edge with a Microsoft Azure Active Directory (AAD) account.  Workspaces are not available when using Guest mode or InPrivate browsing. 

Reviewer: ```suggestion
Microsoft Edge Workspaces is only available for enterprise customers who are signed into Microsoft Edge with a Microsoft Azure Active Directory (AAD) account.  Workspaces aren't available when using Guest mode or InPrivate browsing. 
```
prefer contractions except where emphasis is needed
https://learn.microsoft.com/en-us/style-guide/word-choice/use-contractions
